.travis.yml

```diff
@@ -15,12 +13,21 @@ python:
 
 matrix:
     include:
-        - python: 3.6
+        # Enable 3.7 without globally enabling sudo and dist: xenial for other build jobs
+        - python: 3.7
+          dist: xenial
+          sudo: true
+        - python: 3.7
           env: STYLE='code'
-        - python: 3.6
+          dist: xenial
+          sudo: true
+        - python: 3.7
           env: STYLE='docs'
+          dist: xenial
+          sudo: true
```

Comment:
You can drop `sudo: true`. It is no longer necessary.

Reply:
Just removed it and sent in a new commit.